src/assets/cru-scss/_global-website-list.scss

```diff
@@ -1,4 +1,23 @@
+/***** TESTING *****/
```

Comment:
I agree, this should be cleaned up.  @j2trumpet Can you give input on this.  What lines can be removed?